ios/Classes/TiMapCustomAnnotationView.m

```diff
@@ -24,8 +24,8 @@ - (id)initWithAnnotation:(id <MKAnnotation>)annotation reuseIdentifier:(NSString
 - (void)setProxy:(TiViewProxy*)customView
 {
     if (theProxy != customView) {
-        [[theProxy view] removeFromSuperview];
-        RELEASE_TO_NIL(theProxy);
+        //[[theProxy view] removeFromSuperview];
+        //RELEASE_TO_NIL(theProxy);
```

Comment:
Please always remove it completely, don't comment it out. And I'm afraid this could lead to memory-leaks, have you tested this with the analyser?